how-to-guides/apache-to-minio.md

```diff
@@ -4,17 +4,28 @@ This article explains how to use Fluentd to aggregate and transport Apache logs
 
 ## Prerequisites
 
-The following services are required to be set up correctly:
+The following software/services are required to be set up correctly:
 
-* [Fluentd](../installation/)
-* [Minio](https://minio.io/download/)
-* [Apache](https://httpd.apache.org/)
+* [Fluentd](https://www.fluentd.org/)
+* [Minio](https://minio.io/download/) (S3 Compatible Storage)
+* [Apache](https://httpd.apache.org/) (with the Combined Log Format)
 
-Also, if you have installed Fluentd through RubyGems \(without `fluent-package`\), please install the [`out_s3`](../output/s3.md) plugin manually:
+For simplicity, this article will describe how to set up a one-node configuration.
+Please install the above prerequisites software on the same node.
```

Comment:
> For simplicity, this article will describe how to set up a one-node configuration.
> Please install the above prerequisites software on the same node.

Do we need this for `apache-to-minio`?
Looks like Minio can exist on the another node.